python/server/auto_generated/qgsserverfeatureid.sip.in

```diff
@@ -0,0 +1,76 @@
+/************************************************************************
```

Comment:
Please do not expose this class to python bindings.

Reply:
Why ? it could be usefull in plugin ?

Reply:
Because I wouldn't consider it stable API. There's hopefully going to be an implementation in core at some point which would mean this API here would disappear or be modified, which makes it unstable.

Reply:
I have removed qgsserverfeatureid from python bindings but the code layout test complains.

How to explicitly exclude this class from API ?

Reply:
```
#define SIP_NO_FILE
```